### ISSUE TITLE
Drop Feeds doesn't work on Firefox 61 (macOS)

### ISSUE THREAD

**Issue by thieleke** [dauphine-dev/drop-feeds#37]
Firefox 61 (https://www.mozilla.org/en-US/firefox/61.0/releasenotes/) added the following change that appears to break Drop Feeds: 

> WebExtensions now run in their own process on MacOS

After upgrading Firefox, Drop Feeds 0.7.13 fails to initialize with the following console error:

> 1530027186495	addons.xpi	WARN	Exception running bootstrap method startup on {42d52ade-6923-4f20-92d1-8db031269cc6}: Error: Out-of-process WebExtensions are not supported with multiple child processes (resource://gre/modules/Extension.jsm:1261:13) JS Stack trace: Extension@Extension.jsm:1261:13
> startup@Extension.jsm:1172:22
> callBootstrapMethod@XPIProvider.jsm:2757:20
> startInstall/<@XPIInstall.jsm:2036:11

**Comment by dauphine-dev** [comment]
Unfortunately I have no  MacOS to look that, and I have no ideas of what the issue, Drop Feeds should run with only process.
Or may be it is because there is one process for the sidebar, one for the action button, and one for the options, and one for each script or html file referenced in manifest.json.
May be can you can try to deactivate the action button in the manifest.json (and other stuffs that launched a html or a js other the side bar), and see what happen ?

**Comment by dauphine-dev** [comment]
For information, I have tested on Firefox 61 / Linux there is no issues.

**Comment by thieleke** [comment]
It works on Windows as well

**Comment by thieleke** [comment]
As a temporary workaround, I found that setting ```extensions.webextensions.remote``` to false (default is true) allows Drop Feeds to initialize on macOS.  I'll try to investigate later...

https://www.ghacks.net/2017/07/11/firefox-nightly-out-of-process-web-extensions/ has some info

**Comment by dauphine-dev** [comment]
@thieleke 
Hello Jeff, this issue is still existing ?

**Comment by thieleke** [comment]
Yes, this is still an issue.  Drop Feeds does not load unless extensions.webextensions.remote is set to False  (verified on Firefox 78.0.2 just now)
